src/App.css

```diff
@@ -0,0 +1,283 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --azure: #0290ff;
+  --white: white;
+  --white2: #e8e8e8;
+  --white3: #fafafa;
+  --black2: #121212;
+  --cool-blue: #4386bf;
+  --pale-grey: #f5f6fa;
+  --warm-grey: #888;
+  --very-light-pink: #c4c4c4;
+}
+
+body {
+  background-color: var(--pale-grey);
+  font-family: Montserrat, sans-serif;
+}
+
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) for this project, " `Your final design should match Zeplin's design` ", please kindly ensure your styling is `EXACTLY` the same as the one provided on [Zeplin](https://zeplin.io/). Checkout the screenshots below and kindly fix that. 

***The Design***
![bookStoreDesign](https://user-images.githubusercontent.com/65852621/179352135-4497c257-bd51-4cc0-8c43-1a260035bba8.png)

***Your Implementation***
![bookStoreImplementation](https://user-images.githubusercontent.com/65852621/179352146-38f8ac84-43c6-4b17-be94-3d2d790a21c8.png)